example/shoot-cert-service/base/kustomization.yaml

```diff
@@ -19,3 +19,12 @@ patches:
         type: shoot-cert-service
         workerlessSupported: true
         globallyEnabled: true
+    - op: add
+      path: /spec/resources/1
+      value:
+        kind: Extension
+        type: controlplane-cert-service
+        globallyEnabled: true
```

Comment:
```suggestion
```

`globallyEnabled` is only relevant for shoots. Instead, `autoEnable` can be used when the feature is released (see https://github.com/gardener/gardener/pull/11982).